app/views/layouts/_navigation_links.html.erb

```diff
@@ -6,18 +6,18 @@
     <% if ENV['CM_RESOURCES_URL'] %>
       <li><%= link_to 'CM Resources', ENV['CM_RESOURCES_URL'], target: '_blank' %></li>
     <% end %>
-  <% end %>  
-</ul>    
```

Comment:
Love to see the clean up, ftr. Maybe just make comments in the future akin to that "Did some cleanup of files while I was at it" otherwise you run the risk of having a reviewer go "WHY DID YOU REMOVE SPACES AND TABS?!?!"